theme.json

```diff
@@ -0,0 +1,413 @@
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "16px"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "18px"
+        }
+      },
+      "heading": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.07rem, calc(1.02rem + 0.25vw), 1.28rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.14rem, calc(1.04rem + 0.47vw), 1.53rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "3": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.30rem, calc(1.08rem + 1.09vw), 2.20rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "4": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.38rem, calc(1.08rem + 1.52vw), 2.64rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "5": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
+        "padding": {
+          "global": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
+        },
+        "block": {
+          "gap": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
+        }
+      },
+      "alignment": {
+        "center": "744px",
+        "wide": "1008px"
+      },
+      "background": {
+        "primary": "#f5f2f1"
+      },
+      "text": {
+        "primary": "#13191e",
+        "secondary": "#444140",
+        "helper": "#6a6766",
+        "interactive": "#9a6014"
+      },
+      "link": {
+        "default": "#9a6014",
+        "hover": "#7a4909"
+      },
+      "button": {
+        "primary": {
+          "default": "#13191e",
+          "hover": "#d8a45f",
+          "active": "#ba7920",
+          "label": "#fcfafa"
+        }
+      }
+    },
+    "layout": {
+      "contentSize": "var(--wp--custom--alignment--center)",
+      "wideSize": "var(--wp--custom--alignment--wide)"
+    },
+    "spacing": {
```

Comment:
I turned off all the margin and padding in the original theme but I think they're usable on a block level now. It's better than the block style method I was using to apply padding to blocks. Worth checking out. In that scenario we'd want to enumerate the spacing value options like we do with color palettes or type sizes. That way the user only selects from system tokens but they can use the core spacing UI to do it and it get's applied without taking up your block style slot.